HappyHobby/HappyHobby
# Fixed new account creation bug

To test

* Create a new account using the generic email signup
* If it works properly interests should be loaded shortly or a configure your profile link will link you to the settings page
* Any loading screens should go away shortly after a small period of time or a page refresh
